setup.py

```diff
@@ -48,12 +50,16 @@
    Check that you can install it in a virtualenv by running:
    pip install -i https://testpypi.python.org/pypi transformers
 
-8. Upload the final version to actual pypi:
+   Check you can run the following commands:
+   python -c "from transformers import pipeline; classifier = pipeline('text-classification'); print(classifier('What a nice release'))" 
+   python -c "from transformers import *"
+
+9. Upload the final version to actual pypi:
    twine upload dist/* -r pypi
 
-9. Copy the release notes from RELEASE.md to the tag in github once everything is looking hunky-dory.
+10. Copy the release notes from RELEASE.md to the tag in github once everything is looking hunky-dory.
 
-10. Run `make post-release` (or `make post-patch` for a patch release).
+11. Run `make post-release` (or `make post-patch` for a patch release).
```

Comment:
For `post-patch` it actually needs to be checked out on `master`, would be nice to mention it